src/components/hooks/HookDetails.tsx

```diff
@@ -77,26 +77,20 @@ function HookDetails(props: Props, context) {
   let prettyHookArguments = JSON.stringify(hookArguments, null, 2);
 
   // Extract hook-specific data
-  let targetName;
-  let targetState;
-  let navigateToAllHooks;
-
-  switch (hook.name) {
-    case 'on_task':
-      targetName = 'Task';
-      targetState = hookArguments[0].payload.data.task.status;
-      navigateToAllHooks = e => navigateTask(history, e, hook.task.id, true);
-      break;
-    case 'on_build':
-      targetName = 'Build';
-      targetState = hookArguments[0].payload.data.build.status;
-      navigateToAllHooks = e => navigateBuild(history, e, hook.build.id, true);
-      break;
-    default:
-      targetName = 'Unsupported';
-      targetState = 'UNSUPPORTED';
-      navigateToAllHooks = () => alert('Unsupported hook ' + hook.name);
-      break;
+  let targetName = 'Unsupported';
+  let targetState = 'UNSUPPORTED';
+  let navigateToAllHooks: MouseEventHandler = _ => alert('Unsupported hook ' + hook.name);
```

Comment:
Using alert looks a bit unprofessional, maybe use a material-ui pop-out (modal I think its called) instead?  
Not sure if it is practical from a code perspective.  Just my opinion from a design perspective.

Reply:
This is somewhat an exceptional case and shouldn't normally happen at all.